docs/_includes/common/filter-menu.html

```diff
@@ -0,0 +1,472 @@
+                    <span class="checkbox__icon" hidden>
+                        <svg class="checkbox__unchecked" focusable="false" height="18" width="18">
+                            <use xlink:href="#icon-checkbox-unchecked"></use>
+                        </svg>
+                        <svg class="checkbox__checked" focusable="false" height="18" width="18">
+                            <use xlink:href="#icon-checkbox-checked"></use>
+                        </svg>
+                    </span>
+                </span>
+                <span>Item 1 with a long string that will wrap to another line</span>
+            </div>
+            <div class="filter-menu__item" role="menuitem">
+                <span class="checkbox">
+                    <input aria-label="filter menu checkbox example" class="checkbox__control" type="checkbox" name="filter-menu-checkbox-item-2" tabindex="-1" />
+                    <span class="checkbox__icon" hidden>
+                        <svg class="checkbox__unchecked" focusable="false" height="18" width="18">
+                            <use xlink:href="#icon-checkbox-unchecked"></use>
+                        </svg>
+                        <svg class="checkbox__checked" focusable="false" height="18" width="18">
+                            <use xlink:href="#icon-checkbox-checked"></use>
+                        </svg>
+                    </span>
+                </span>
+                <span>Item 2</span>
+            </div>
+            <div class="filter-menu__item" role="menuitem">
+                <span class="checkbox">
+                    <input aria-label="filter menu checkbox example" class="checkbox__control" type="checkbox" name="filter-menu-checkbox-item-3" tabindex="-1" />
+                    <span class="checkbox__icon" hidden>
+                        <svg class="checkbox__unchecked" focusable="false" height="18" width="18">
+                            <use xlink:href="#icon-checkbox-unchecked"></use>
+                        </svg>
+                        <svg class="checkbox__checked" focusable="false" height="18" width="18">
+                            <use xlink:href="#icon-checkbox-checked"></use>
+                        </svg>
+                    </span>
+                </span>
+                <span>Item 3</span>
+            </div>
+        </div>
+    </div>
+</span>
+    {% endhighlight %}
+    </div>
+
+    <h3 id="filter-menu-footer">Filter Menu with Footer Option</h3>
+    <p>An optional footer is available for actions which can go at the bottom of the filter menu, such as <em>Clear all</em> or <em>Select all</em>.</p>
+
+    <div class="demo">
+        <div class="demo__inner">
+            <span class="filter-menu">
+                <div class="filter-menu__items" role="menu">
+                    <div class="filter-menu__scroll-area">
```

Comment:
Since you use `filter-menu__item--footer` would `filter-menu__item--body` be more standard?

Reply:
actually... we should probably drop "item" and just have it:
filter-menu__body
filter-menu__footer

That would be better BEM I think

Reply:
Oh. I see now. The footer is actually being treated as menu item too.  Is that for a11y?  This still seems odd somehow.

Reply:
Yes, the footer is a menu item, too. Although, because we're using the aria-role we can change the class names to whatever we want.

Reply:
The footer should not be a menu item. It's just a button.

Reply:
If it's a pain to rework it, it could stay as a menuitem, but the markup might prove to be a bit of a problem in terms of the menu a11y semantics.

Reply:
The style was easy to fix. Is it a problem to have the button inside an `aria-role="menu"`?

Reply:
Yes, we can't have a button inside `role=menu`.